### PR TITLE
[Feat] Add Docker support

### PR DESCRIPTION
**Pull request by 20uf**
Hello,

You can use docker in bash mode to use Certificationy Cli.

1) Download, build and run Docker image
```
$ docker-compose up -d
```

2) Ssh into local docker instance
```
$ sh docker-ssh
```

3) Running CLI Certificationy 
```
$ php certificationy.php
```
**Small preview**
certificationy_docker

4) To exit bash docker
```
$ exit
 ```

Enjoy !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,17 @@ $ composer create-project certificationy/certificationy-cli
 $ php certificationy.php
 ```
 
+##  Using with Docker
```
Comment (mickaelandrieu) [inline README.md]: With Docker and Docker compose

**Comment by mickaelandrieu** [comment]
I need to do some tests on Windows, I let you know asap

Thank you for this pull request I'm definitely in favor of docker integration!

**Comment by 20uf** [comment]
Do you have any feedback on using windows?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: '2'
+
+services:
+  php:
+    build: .
+    environment:
+      SYMFONY_ENV: dev
+    volumes:
+      - ./:/srv/certificationy
+      - ./vendor:/srv/certificationy/vendor
```
Comment (roukmoute) [inline docker-compose.yml]: Is it usefull ?

Comment (20uf) [inline docker-compose.yml]: Of course it is necessary to mount the files on the image.

Comment (roukmoute) [inline docker-compose.yml]: Yes of course, but if you have already `./:/srv/certificationy`, `./vendor:/srv/certificationy/vendor` is automatically included no?

Comment (20uf) [inline docker-compose.yml]: Yes indeed I had misunderstood, a line is in too much :)

Comment (20uf) [inline docker-compose.yml]: Fixed.

**Comment by mickaelandrieu** [comment]
Hello @20uf, totally forgot this pull request :/

Let's merge it, if I had feedbacks I'll suggest a pull request to improve it :)

Thanks for your contribution!

**Pull request merged**